inst/templates/core/01_prepr.R

```diff
@@ -4,14 +4,16 @@ project_name <- "{{ aoi_name }}"
 
 fid <- read_fid()
 
-aoi <- snap_aoi()
+#aoi <- snap_aoi("datecreek_aoi.gpkg")
+# I had some issues with just using the name as not able to detect file in location
```

Comment:
Yeah, you do need to provide the path... or leave blank and it will try to find it in the default location.

Reply:
I think I was also finding the snap_aoi() was not able to locate the file when present and the parentheses were blank, ie  your internal look_for_aoi() was not finding the correct root dir, but I will do some more testing on that first